tensorboard/plugins/mesh_visualizer/BUILD

```diff
@@ -0,0 +1,116 @@
+exports_files(["LICENSE"])
+
+load("//tensorboard/defs:protos.bzl", "tb_proto_library")
+
+py_library(
+    name = "metadata",
+    srcs = ["metadata.py"],
+    srcs_version = "PY2AND3",
+    deps = [
+        ":protos_all_py_pb2",
+        "//tensorboard/compat/proto:protos_all_py_pb2",
+    ],
+)
+
+py_test(
+    name = "metadata_test",
+    size = "small",
+    srcs = ["metadata_test.py"],
+    srcs_version = "PY2AND3",
+    deps = [
+        ":metadata",        
+        "//tensorboard:expect_tensorflow_installed",
+        "//tensorboard/util:test_util",
+        "@org_pythonhosted_six",
+    ],
+)
+
+py_library(
+    name = "mesh_plugin",
+    srcs = ["mesh_plugin.py"],
+    srcs_version = "PY2AND3",
+    visibility = ["//visibility:public"],
+    deps = [
+        ":metadata",
+        ":protos_all_py_pb2",
+        "//tensorboard:expect_numpy_installed",        
+        "//tensorboard:plugin_util",
+        "//tensorboard/backend:http_util",
+        "//tensorboard/backend/event_processing:event_accumulator",        
+        "//tensorboard/plugins:base_plugin",
+        "//tensorboard/util:tensor_util",
+        "@org_pythonhosted_six",
+        "@org_pocoo_werkzeug",
+    ],
+)
+
+py_library(
+    name = "test_utils",
+    testonly = 1,
+    srcs = ["test_utils.py"],
+    srcs_version = "PY2AND3",
+    deps = [
+        ":mesh_summary",
+        "//tensorboard:expect_tensorflow_installed",        
+        "//tensorboard/backend:application",
+        "//tensorboard/backend/event_processing:event_multiplexer",
+        "@org_pocoo_werkzeug",
+    ],
+)
+
+py_test(
+    name = "mesh_plugin_test",
+    size = "small",
+    srcs = ["mesh_plugin_test.py"],
+    srcs_version = "PY2AND3",
+    deps = [
+        ":mesh_plugin",
+        ":mesh_summary",
+        ":test_utils",
+        "//tensorboard:expect_numpy_installed",        
+        "//tensorboard:expect_tensorflow_installed",        
+        "//tensorboard/backend:application",
+        "//tensorboard/backend/event_processing:event_multiplexer",
+        "//tensorboard/plugins:base_plugin",
+        "//tensorboard/util:test_util",
+        "@org_pocoo_werkzeug",
+        "@org_pythonhosted_six",
+        "@org_pythonhosted_mock",
+    ],
+)
+
+py_library(
+    name = "mesh_summary",
+    srcs = ["mesh_summary.py"],
+    srcs_version = "PY2AND3",
+    visibility = [
+        "//visibility:public",
+    ],
+    deps = [
+        ":metadata",
+        ":protos_all_py_pb2",
+        "//tensorboard/compat:tensorflow",
+    ],
+)
+
+py_test(
+    name = "mesh_summary_test",
+    size = "small",
+    srcs = ["mesh_summary_test.py"],
+    srcs_version = "PY2AND3",
+    deps = [
+        ":mesh_summary",
+        ":test_utils",        
+        "//tensorboard:expect_tensorflow_installed",
+        "//tensorboard/util:test_util",
+    ],
+)
+
+tb_proto_library(
+    name = "protos_all",
+    srcs = ["plugin_data.proto"],
+    visibility = ["//visibility:public"],
+)
```

Comment:
Please fix missing newlines at end of file. For this file, you can just
run `buildifier`.

Reply:
I don't think I have buildifier on my mac, should i just pip install it?

Reply:
It’s from <https://github.com/bazelbuild/buildtools>; it looks like you
can install it from:
<https://github.com/bazelbuild/buildtools/releases/download/0.25.0/buildifier.mac>

Reply:
how do you install .mac files?

Reply:
Just run it:

```
$ wget -q 'https://github.com/bazelbuild/buildtools/releases/download/0.25.0/buildifier.mac'
$ file buildifier.mac 
buildifier.mac: Mach-O 64-bit x86_64 executable
```